src/components/HighTable/HighTable.module.css

```diff
@@ -49,15 +49,20 @@
   }
   th[aria-sort="ascending"]::after,
   th[aria-sort="descending"]::after {
-    position: absolute;
-    right: 0;
-    top: 0;
+    position: relative;
+    display: inline-block;
+    margin-left: 12px;
+    font-size: 14px;
```

Comment:
normally would say don't use px for font-size. This is effectively an icon so I think it's okay. But if someone scale up text, should the sort indicator scale with the text?

PS- I looks really nice!